main.py

```diff
@@ -0,0 +1,57 @@
+import random
+from replit import clear 
+from hangman_art import stages, logo
+from hangman_words import word_list
+
+print(logo)
+chosen_word = random.choice(word_list)
+lives = 6
+
+#Testing code
+print(f'Pssst, the solution is {chosen_word}.')
+
+#Create blanks
+display = ["_" for _ in range(len(chosen_word))]
+#Displey blanks
+print(display)
+
+game_on = True
+guesses_letters = ''
+
+while game_on:
+    
+    guess = input("Guess a letter: \n").lower()
+    clear()
+    #Printing ifromation if letter has alredy chosen
+    if guess in guesses_letters:
+        print(f"Letter {guess} has already chosen")
+    
+    #Check guessed letter
+    for ID_letter in range(len(chosen_word)):
+
+        if chosen_word[ID_letter] == guess:
+            display[ID_letter] = guess
+    
+    #Adding gueses leetter to string with the letters
+    guesses_letters += guess
+            
+    #Check if user is wrong.
+    if guess not in chosen_word:
+        print(f'Letter "{guess}" is not in the word.')
```

Comment:
Try to use the same style across strings, either stay with double-quote or single apostrophe.

Reply:
ok :)